### PR TITLE
Upgrade V8 to 7.4.98 (kKeep fix)

### PR DESCRIPTION
**Pull request by ry**
Fixes https://github.com/denoland/deno/issues/814
https://bugs.chromium.org/p/v8/issues/detail?id=8761

### CODE REVIEW

File: tests/error_005_missing_dynamic_import.ts.out [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-    at resolveModuleNamesWorker ([WILDCARD])
+[WILDCARD]NotFound: Cannot resolve module "bad-module.ts" from "[WILDCARD]tests/error_005_missing_dynamic_import.ts"
+    at DenoError (deno/js/errors.ts:[WILDCARD])
+    at maybeError (deno/js/errors.ts:[WILDCARD])
+    at maybeThrowError (deno/js/errors.ts:[WILDCARD])
+    at sendSync (deno/js/dispatch.ts:[WILDCARD])
+    at codeFetch (deno/js/os.ts:[WILDCARD])
+    at resolveModule (deno/js/compiler.ts:[WILDCARD])
+    at deno/js/compiler.ts:[WILDCARD]
+    at resolveModuleNames (deno/js/compiler.ts:[WILDCARD])
```
Comment (hayd) [inline tests/error_005_missing_dynamic_import.ts.out]: wasn't the [WILDCARD] here to support tests which don't have deno in a directory called deno?

Comment (ry) [reply tests/error_005_missing_dynamic_import.ts.out]: Good catch - fixed.

**Comment by ry** [comment]
Basically no speed up, unfortunately:
```
> ./prebuilt/linux64/hyperfine --warmup 3 "./target/release/deno  tests/003_relative_import.ts --recompile" "deno tests/003_relative_import.ts --recompile"
Benchmark #1: ./target/release/deno  tests/003_relative_import.ts --recompile
  Time (mean ± σ):     384.1 ms ±   4.8 ms    [User: 590.8 ms, System: 34.8 ms]
  Range (min … max):   380.1 ms … 396.3 ms    10 runs

Benchmark #2: deno tests/003_relative_import.ts --recompile
  Time (mean ± σ):     412.6 ms ±   3.6 ms    [User: 622.8 ms, System: 32.8 ms]
  Range (min … max):   407.2 ms … 419.1 ms    10 runs

Summary
  './target/release/deno  tests/003_relative_import.ts --recompile' ran
    1.07 ± 0.02 times faster than 'deno tests/003_relative_import.ts --recompile'
```

**Comment by kitsonk** [comment]
> Probably we need to exercise the TS compiler before snapshotting to get optimized code.

I looked at it before, it is semi tricky, we might have to do something silly like actually compile some code and emit it.

**Comment by ry** [comment]
Fix has been landed in V8 7.4.98.

This patch is ready to land.

File: tools/third_party.py [hunk 2]
```diff
@@ -255,8 +255,7 @@ def download_clang_format():
 
 # Download clang by calling the clang update script.
 def download_clang():
-    run(['python',
-         tp('v8/tools/clang/scripts/update.py'), '--if-needed'],
+    run(['python', tp('v8/tools/clang/scripts/update.py')],
```
Comment (piscisaureus) [inline tools/third_party.py]: No more  --if-needed?

Comment (ry) [reply tools/third_party.py]: Dont think so / they seem to have removed it.

**Pull request merged**